streamSignupManager/README.md

```diff
@@ -0,0 +1,112 @@
+# RSCBot: combineRooms
```

Comment:
This should be streamSignupManager instead of combineRooms, correct?

Reply:
yes, thank you!